TypeCobol/Compiler/Text/ColumnsLayout.cs

```diff
@@ -15,6 +15,7 @@ public enum ColumnsLayout
         /// Columns 73-> : Comment
         /// </summary>
         CobolReferenceFormat,
+        CobolReferenceFormatWithoutCommentText,
```

Comment:
If it is only dedicated to ContinuationLine, maybe we could rename it into `ContinuationLineFormat `?
I will discuss this with @fm-117

Reply:
@efr15 I suggest `CobolReferenceFormatForContinuationLine`

Reply:
`CobolReferenceFormatUnlimitedLength` ? I'm also ok with `CobolReferenceFormatForContinuationLine`. At any rate, add a comment on this special enum value.

Reply:
CobolReferenceFormatUnlimitedLength + comment added